src/api/app/views/webui2/webui/request/show.html.haml

```diff
@@ -1,15 +1,27 @@
+- content_for(:content_for_head, javascript_include_tag('webui2/cm2/index-diff'))
+
+- if @not_full_diff
+  = render partial: 'webui2/shared/truncated_diff_hint',
+  locals: { path: request_show_path(number: @number, full_diff: true) }
+
 .card.mb-3
   .card-header.d-flex.justify-content-between
     %h5
       Overview
   .card-body
     -# description content
 .card.mb-3
-  .card-header.d-flex.justify-content-between
-    %h5
-      Diff
+  .bg-light
+    %ul.nav.nav-tabs{ role: 'tablist' }
+      - @actions.each_with_index do |action, index|
+        %li.nav-item
+          = link_to(action[:name], "##{valid_xml_id(action[:name])}", class: "nav-link text-nowrap #{('active' if index == 0)}",
```

Comment:
`index.zero?` (if not you are introducing a Rubocop offense, Rubocop doesn't complain because we have many offenses, but the cop is enabled and agreed)